kedro/config/omegaconf_config.py

```diff
@@ -136,10 +138,20 @@ def __init__(  # noqa: too-many-arguments
             env=env,
             runtime_params=runtime_params,
         )
+        # Read globals from patterns if the files exist
+        try:
+            self.globals = self._get_config("globals")
```

Comment:
What happens if a user tries to call `config_loaderr["globals"]`?

Reply:
In the code if user does `config_loader["globals"]` it would return an empty dict `{}`

Reply:
It's not throwing a `MissingConfigException` because we don't require users to have `globals.yml` in their config necessarily

Reply:
Will it also return an empty dict if the user does have globals? That would be quite confusing..